--- a/chrome/browser/resources/settings/site_settings/v8_page.ts
+++ b/chrome/browser/resources/settings/site_settings/v8_page.ts
@@ -56,7 +56,7 @@ export class V8PageElement extends V8Pag
   }
 
   private getBlockForUnfamiliarSitesSubLabel_(): string {
-    const safeBrowsingSetting = this.getPref('generated.safe_browsing').value;
+    const safeBrowsingSetting = SafeBrowsingSetting.DISABLED;
     return this.i18n(
         safeBrowsingSetting === SafeBrowsingSetting.DISABLED
             ? 'siteSettingsJavascriptOptimizerBlockedUnfamiliarSitesSafeBrowsingOffSubLabel'
